### PR TITLE
Changed periods in the script templates

### PR DESCRIPTION
**Pull request by YeldhamDev**
Comments in the docs don't end with a period, unless it's to start another phrase. Made so that the script templates follow suit, which also makes them consistent with the comments right above them.

Also, clang-format requested to add a tab to a comment in the script.

**EDIT**: Nevermind, I guess something is up to the clang-format in my end then.

### CODE REVIEW

**Comment by groud** [comment]
> Comments in the docs don't end with a period, unless it's to start another phrase.

I have no clue where you found this rule. Is there any document that states that ?

**Comment by YeldhamDev** [comment]
@groud A written out rule? No. But the vast majority (excluding very few exceptions) of comments are written this way.

**Comment by vnen** [comment]
> [...] also makes them consistent with the comments right above them.

I believe this alone is a good reason to accept the change.

**Comment by mhilbrunner** [comment]
I don't get this change. @vnen:
>[...] also makes them consistent with the comments right above them.

But the comments in the lines immediately above the ones that are changed in the diff **do** end with periods.

**Comment by vnen** [comment]
@mhilbrunner I mean the last comment in the `_ready()` function. It is "`# Initialization here`" without a period.

There's also the options to end everything with periods, or remove them all.

**Comment by mhilbrunner** [comment]
IMO, either end all lines with periods or none.
`Called every frame. Delta is time since last frame.` is going to be problematic, as removing the last period will look ugly. Either break this up into multiple lines (without periods) or go with periods for all lines.

**Comment by volzhs** [comment]
I second end with the period for all lines.

**Comment by YeldhamDev** [comment]
I can add the period for the C# template, but for GDScript, which is supposed to be Python-like, I think it would be better to stick with their style for this: https://www.python.org/dev/peps/pep-0008/#comments

File: modules/gdscript/gdscript_editor.cpp [hunk 1]
```diff
@@ -65,7 +65,7 @@ Ref<Script> GDScriptLanguage::get_template(const String &p_class_name, const Str
 					   "%TS%pass\n\n" +
 					   "#func _process(delta):\n" +
 					   "#%TS%# Called every frame. Delta is time since last frame.\n" +
-					   "#%TS%# Update game logic here.\n" +
+					   "#%TS%# Update game logic here\n" +
```
Comment (akien-mga) [inline modules/gdscript/gdscript_editor.cpp]: That makes no sense, the sentence above ends with a period.
The PEP8 convention is (at least as I understand it and saw it applied):
- single line comment, no period
- multiple lines/multiple sentences: all sentences end with a period

Comment (YeldhamDev) [reply modules/gdscript/gdscript_editor.cpp]: Sorry, I understood the guide incorrectly. I will just add the missing periods. I just wish that some care for the consistency is taken, and the docs (specially the ones inside the editor) follow suit.

**Pull request merged**